python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -216,6 +234,11 @@ def skip_on_parent_outdated() -> "SkipOnParentOutdatedRule":
         """Skip materializing an asset partition if one of its parents is outdated."""
         return SkipOnParentOutdatedRule()
 
+    @staticmethod
+    def skip_on_parent_missing() -> "SkipOnParentMissingRule":
+        """Skip materializing an asset partition if one of its parents is outdated."""
```

Comment:
Is this the right docstring?

Also, I think it's worth going out of our way to be as unambiguous as possible about the behavior in these docstrings. E.g. spell out what happens when there are partitions. What about when there are observable source assets?

Reply:
Good call, updated